### DISCUSSION TITLE
BackTop是否有对齐FloatButton的计划

### DISCUSSION THREAD

**Discussion by sy0uta1k0n** [vueComponent/ant-design-vue#6353] [Ideas]
目前我们业务侧需要抹平react和vue的页面差异. 

在回顶这里碰到点小问题, FloatButton提供了折叠展开功能, BackTop需要自己写一个对齐. 以后是否有计划对齐哇?

antd:
https://ant.design/components/float-button-cn

antdv:
https://www.antdv.com/components/back-top-cn

**Comment by tangjinzhou** [comment]
antdv 4

**Reply by sy0uta1k0n** [reply]
看到了, 谢谢大佬
